plugins/cpp_metrics/service/src/plugin.cpp

```diff
@@ -0,0 +1,33 @@
+#include <webserver/pluginhelper.h>
+
+#include <service/cppmetricsservice.h>
+
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wreturn-type-c-linkage"
+extern "C"
+{
+  boost::program_options::options_description getOptions()
+  {
+    namespace po = boost::program_options;
+
+    po::options_description description("Cpp Metrics Plugin");
+
+    description.add_options()
+      ("dummy-result", po::value<std::string>()->default_value("Dummy result"),
+        "This value will be returned by the dummy service.");
+
+    return description;
+  }
```

Comment:
If there are no extra options, this could be simplified to:
```cpp
boost::program_options::options_description getOptions()
{
  return boost::program_options::options_description("Cpp Metrics Plugin");
}
```

Reply:
Fixed in #a904b68c9f6b79f09bb04a1a1461cec0d803af9e.